modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -568,7 +568,17 @@ public ObjectField updateCustomObjectField(
 		_setBusinessTypeAndDBType(businessType, dbType, newObjectField);
 
 		newObjectField.setListTypeDefinitionId(listTypeDefinitionId);
-		newObjectField.setDBColumnName(name + StringPool.UNDERLINE);
+
+		if (StringUtil.equals(
+				newObjectField.getBusinessType(),
+				ObjectFieldConstants.BUSINESS_TYPE_RELATIONSHIP)) {
+
+			newObjectField.setDBColumnName(name);
+		}
+		else {
+			newObjectField.setDBColumnName(name + StringPool.UNDERLINE);
+		}
```

Comment:
Hi @p-albuquerque, could you explain a little more why we can't have the underline for relationship fields?

Don't we need to add an upgrade process to fix the existing columns?

Thanks!